lib/chef/audit/runner.rb

```diff
@@ -96,6 +97,8 @@ def configure_rspec
         RSpec.configure do |c|
           c.color = Chef::Config[:color]
           c.expose_dsl_globally = false
+          separator = File::ALT_SEPARATOR ? File::ALT_SEPARATOR : File::SEPARATOR
+          c.backtrace_exclusion_patterns << Regexp.new(__FILE__.split(separator)[0..2].join(separator))
```

Comment:
on windows isn't this going to give you `%w{c: opscode chef}` while on unix you'll get `%w{opt chef embedded}` and on my rvm install you'll get `%w{home lamont .rvm}`, etc?  You probably want a `-2` on the ending range or something like that?